### PR TITLE
Add mirrored copy of blog article “Introducing sig-etcd”

### PR DESCRIPTION
**Pull request by wenjiaswe**
Thanks to @logicalhan @serathius for the effort of making etcd a Kubernetes special interest group, @fsmunoz for helping putting together the doc, and @jmhbnz for posting the blog in etcd.

This post is to celebrate introducing etcd as a k8s sig.

cc @ahrtr @jberkus @kaslin @ptabor @mitake

### CODE REVIEW

**Comment by nate-double-u** [comment]
This is a mirror of https://etcd.io/blog/2023/introducing-sig-etcd/

Usually we like to sync them, but I think we may have missed the boat on that. Are they otherwise identical?

**Comment by nate-double-u** [comment]
We’ll at least need to change the publication date to a future date

/hold

**Comment by nate-double-u** [comment]
Nov 6 may be  a good date, will give us some time to review it.
Board https://github.com/orgs/kubernetes/projects/93/views/3

**Comment by wenjiaswe** [comment]
> Nov 6 may be a good date, will give us some time to review it. Board https://github.com/orgs/kubernetes/projects/93/views/3

Thanks, updated.

**Comment by wenjiaswe** [comment]
> This is a mirror of https://etcd.io/blog/2023/introducing-sig-etcd/
> 
> Usually we like to sync them, but I think we may have missed the boat on that. Are they otherwise identical?

Yes, this is the exact the same as the etcd blogpost https://etcd.io/blog/2023/introducing-sig-etcd/. Thanks for the reminding, we will do the sync instead next time.

**Comment by nate-double-u** [comment]
Sorry, i should have mentioned earlier, could you also add a canonical URL field in the front matter as well?
See these upcoming blog entries for the format:
* https://github.com/kubernetes/website/pull/43775/files
* https://github.com/kubernetes/website/pull/43474/files

**Comment by wenjiaswe** [comment]
> Sorry, i should have mentioned earlier, could you also add a canonical URL field in the front matter as well? See these upcoming blog entries for the format:
> 
> * https://github.com/kubernetes/website/pull/43775/files
> * https://github.com/kubernetes/website/pull/43474/files

Thanks for reminding. Added.

File: content/en/blog/_posts/2023-11-06-introducing-sig-etcd.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+layout: blog
+title: "Introducing sig-etcd"
+slug: introducing-sig-etcd
+date: 2023-11-06
+canonicalUrl: https://www.k8s.dev/blog/2023/11/06/introducing-sig-etcd/
```
Comment (nate-double-u) [inline content/en/blog/_posts/2023-11-06-introducing-sig-etcd.md:6]: should be https://etcd.io/blog/2023/introducing-sig-etcd/, not k8s.dev

```suggestion
canonicalurl: https://etcd.io/blog/2023/introducing-sig-etcd/
```

**Comment by sftim** [comment]
/hold

We usually avoid mirroring articles that appear more than 24 hours after initial publication.
With this not having been planned in, I think we should stick to that convention and _not_ mirror the article here.

**Comment by sftim** [comment]
Maybe we can have a (different) post-kubecon article that wraps up recent announcement and hyperlinks to the SIG etcd article?

**Comment by liangyuanpeng** [comment]
cc

**Comment by sftim** [comment]
/retitle Add mirrored copy of blog article “Introducing sig-etcd”

I don't think we'll merge this as-is @wenjiaswe; we'd have been happy to mirror the article with notice ahead of publication, but that didn't happen.

**Comment by sftim** [comment]
I'm going to close this PR, as we wouldn't merge it.

Thanks for the proposal @wenjiaswe; it's unfortunate that the timing wasn't right.

/close

**Comment by jberkus** [comment]
@sftim I'm going to ask you to reconsider.  It's a new SIG, being announced here at Kubecon; it would be valuable to have content on the Kubernetes blog in a more timely fashion.  It's one thing if there's content that's not appropriate for the Kubernetes blog, or if nobody has time to review.  But if the only problem is that they missed the mirroring window by 24 hours, that seems like "rules over spirit".

**Comment by nate-double-u** [comment]
I agree with @jberkus here. We have a new SIG and this is the opportunity to get the most eyes on it. Posting it after KubeCon will not be nearly as impactful as getting it up before or (now,) during.

**Comment by nate-double-u** [comment]
Reopening to facilitate conversation.

/reopen

**Comment by nate-double-u** [comment]
Holding for publishing decision
/hold

**Comment by divya-mohan0209** [comment]
While I agree with Tim that this isn't per the norm and should have, ideally, been submitted earlier for review and approval. However, taking into cognizance that this is a new SIG that is being introduced at KubeCon we should consider making an exception.

**Comment by fsmunoz** [comment]
Hello,

As co-author of the article I would just like to add that I was under the assumption that it would be published here, and was unaware that it was planned to be published elsewhere - not that I have anything against it! I would have tried to make things sync better otherwise.

**Comment by wenjiaswe** [comment]
Thanks everyone for chiming in. I apologize,  being a new SIG, there is indeed a bit gap there. We weren't aware of this 24hours mirroring policy, will definitely follow the rule in future.

**Comment by reylejano** [comment]
Ideally there would have been better coordination efforts to mirror this blog article. In my opinion, we can make an exception in this case with the timing of the introduction of SIG etcd at KubeCon and publishing a mirror of this blog that introduces SIG etcd

/lgtm

**Comment by nate-double-u** [comment]
Updated to remove commit error.

**Comment by natalisucks** [comment]
/approve
The SIG Docs leads have been able to have conversations with @wenjiaswe and @nate-double-u about process, and while we've already commented about the process preferably being better followed in future, we agree that it's worthwhile publishing due to contributor wrangling for etcd, specifically at KubeCon NA.

@sftim We want you to have the final call here. If you're amenable to this decision, it would be great for you to remove the hold in order for the blog to publish by Tuesday. A huge thanks in advance 🙏

File: content/en/blog/_posts/2023-11-06-introducing-sig-etcd.md
Comment (sftim) [inline content/en/blog/_posts/2023-11-06-introducing-sig-etcd.md:1]: Post-merge, please fix the path to `content/en/blog/_posts/2023-11-07-introducing-sig-etcd.md`

File: content/en/blog/_posts/2023-11-06-introducing-sig-etcd.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+layout: blog
+title: "Introducing sig-etcd"
+slug: introducing-sig-etcd
+date: 2023-11-07
+canonicalUrl: https://etcd.io/blog/2023/introducing-sig-etcd/
+---
+
+**Author**:  "[Han Kang](https://github.com/logicalhan) (Google), [Marek Siarkowicz](https://github.com/serathius) (Google), [Frederico Muñoz](https://github.com/fsmunoz) (SAS Institute)"
```
Comment (sftim) [inline content/en/blog/_posts/2023-11-06-introducing-sig-etcd.md:9]:
```suggestion
**Authors**:  Han Kang (Google), Marek Siarkowicz (Google), Frederico Muñoz (SAS Institute)
```

**Review by sftim — COMMENTED** [review]
I (and I think we) don't want people's RSS feeds to “go back in time”. However, this article's got today's date on it. That's fine.

I think we can publish this. I'll tweak it first.

We should definitely continue to avoid adding “new” articles that appear retrospectively, making rare exceptions there.

**Review by sftim — COMMENTED** [review]
/hold cancel

**Comment by nate-double-u** [comment]
Thanks everyone!

**Pull request merged**